src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -412,6 +467,30 @@ public function getDigitisedBy()
     }
 
     /**
+     * Get the last time a user edited the track.
+     *
+     * @return bool
+     */
+    public function getLastEdited()
+    {
+        return $this->lastedited;
+    }
+
+    /**
+     * Get the user who last edited the track.
+     *
+     * @return MyRadio_User $lasteditedby The user who last edited the track.
+     */
+    public function getLastEditedBy()
+    {
+        if ($this->lasteditedby === null) {
+            return;
+        } else {
+            return MyRadio_User::getInstance($this->lastedited);
```

Comment:
Isn't this the timestamp? I would recommend using clearer variable names, like last_edited_time and last_edited_memberid